packages/lib/src/spatial-navigation/components/ScrollView.tsx

```diff
@@ -59,9 +152,80 @@ export const SpatialNavigationScrollView = ({
         showsHorizontalScrollIndicator={false}
         showsVerticalScrollIndicator={false}
         scrollEnabled={false}
+        onScroll={(event) => {
+          scrollY.current = event.nativeEvent.contentOffset.y;
+        }}
+        scrollEventThrottle={16}
       >
         {children}
       </ScrollView>
+      {deviceType === 'remotePointer' ? (
+        <PointerScrollArrows
+          topArrow={topArrow}
+          bottomArrow={bottomArrow}
+          topArrowContainerStyle={topArrowContainerStyle}
+          bottomArrowContainerStyle={bottomArrowContainerStyle}
+          onMouseEnterTop={onMouseEnterTop}
```

Comment:
I thought this was supposed to be named onMouseEnterAscending/Descending 🤔 
ScrollViews can be horizontal too, right?

Reply:
Fixed :)